src/main/java/com/foo/unmls/UnmlsGraphTool.kt

```diff
@@ -0,0 +1,41 @@
+package com.foo.unmls
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.core.subcommands
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.required
+import com.github.ajalt.clikt.parameters.types.int
+
+object UnmlsGraphTool {
```

Comment:
An starting point for you to use or discard as you prefer. This is Kotlin code just to give you a feel for what it's like; see the docs at https://kotlinlang.org/ for more detail. An `object` is a class that is automatically a singleton -- just right for the host of a `main()` method.